Framework/Muon/src/PlotAsymmetryByLogValue.cpp

```diff
@@ -206,6 +208,22 @@ void PlotAsymmetryByLogValue::exec() {
   saveResultsToADS(outWS, nplots + 1);
 }
 
+std::string PlotAsymmetryByLogValue::getLogUnits(const std::string fileName) {
+  Workspace_sptr loadedWs = doLoad(fileName);
+  MatrixWorkspace_sptr ws;
+  // Check if workspace is a workspace group
+  WorkspaceGroup_sptr group = std::dynamic_pointer_cast<WorkspaceGroup>(loadedWs);
```

Comment:
We could shorten this a bit to something like
```
// Check if workspace is a workspace group, if it is not, we only have 'red' data
if (WorkspaceGroup_sptr group = std::dynamic_pointer_cast<WorkspaceGroup>(loadedWs)) {
  ws = std::dynamic_pointer_cast<MatrixWorkspace>(group->getItem(m_red - 1));
} else {
  ws = std::dynamic_pointer_cast<MatrixWorkspace>(loadedWs);
}
```
But it is your choice which is more readable for maintainability

Reply:
I think we should leave it to be consistent with the rest of the code.